client/js/components/ecosystems/StopFinder.js

```diff
@@ -2,16 +2,36 @@
 
 const React = require('react');
 const Component = React.Component;
+// const palette = require('./../../palette');
```

Comment:
In general, you shouldn't leave commented-out code in a PR.  If we're not going to use it for now, please remove this line.